app/uk/gov/hmrc/transitmovementsvalidator/config/AppConfig.scala

```diff
@@ -27,6 +27,8 @@ class AppConfig @Inject() (config: Configuration) {
 
   lazy val appName: String = config.get[String]("appName")
 
+  lazy val versionHeaderKey: String = config.get[String]("version.header.key")
```

Comment:
This key is going to be always constant. We should create some constant for this instead of declaring it it conf file. 
Like ```https://github.com/hmrc/common-transit-convention-traders/blob/5c59c431183797ba03e6718c2aba89819f1060a8/app/config/Constants.scala#L32```

Reply:
That's fair, I think I left it configurable cause I was deciding on a name at the time, i'll get this changed